src/github.com/getlantern/flashlight/README.md

```diff
@@ -45,7 +45,7 @@ Usage of flashlight:
   -portmap=0: try to map this port on the firewall to the port on which flashlight is listening, using UPnP or NAT-PMP. If mapping this port fails, flashlight will exit with status code 50
   -registerat="": base URL for peer DNS registry at which to register (e.g. https://peerscanner.getiantem.org)
   -role="": either 'client' or 'server' (required)
-  -server="": FQDN of flashlight server when running in server mode (required)
+  -frontfqdns="": YAML string representing a map from the name of each front provider to a FQDN that will reach this particular server via that provider (e.g. '{cloudflare: fl-001.getiantem.org, cloudfront: blablabla.cloudfront.net}')
```

Comment:
(tl;dr Not a big deal either way, I guess.)

I don't know whether golang will parse those two "words" as the value of `-frontfqdns` without surrounding them with quote marks.  Anyway, we were already including YAML in flashlight (this frontfqdns dict itself will be saved as YAML in the config), it takes one line to parse, it looks less like something that could be confused with `-argument=val` bash syntax, and it's a well-known format so we can describe the format with two words in the help string.